test/common/swaggerSpec.js

```diff
@@ -0,0 +1,171 @@
+var node = require('../node');
+var apiSpec = node.swaggerDef;
+var	_ = require('lodash');
```

Comment:
white space

Reply:
Same here... Don't see this space issue on the code locally.